route/route.go

```diff
@@ -166,6 +187,27 @@ func (r *Router) LnS(incomingOrPeer string) {
 		Handler: muxxer,
 	}
 
+	if len(grpcAddr) > 0 {
+		l, err := net.Listen("tcp", grpcAddr)
+		if err != nil {
+			r.iopLogger.Error().Logf("failed to listen to grpc addr: " + grpcAddr)
```

Comment:
Do we want to return if there's an error here?

Reply:
I think we should fail to start if a grpc address has been set and we're unable to bind to the port. We return if the HTTP server is unable to start. If something is reliant on gRPC running and the process doesn't fail to start, it could look like we're working but would reject gRPC traffic.